src/app/root.js

```diff
@@ -1,9 +1,9 @@
-import produce from "immer";
+import immerProduce from "immer";
```

Comment:
This change was because `immer` also has a named export `produce` which throws an issue with the `import` eslint config package.